style.css

```diff
@@ -0,0 +1,1304 @@
+}
+
+.li-container::before {
+  content: "";
+  width: 100%;
+  height: 1px;
+  background-color: rgba(0, 0, 0, 0.1);
+  position: absolute;
+  bottom: -1rem;
+  left: 0;
+}
+
+/* ----------------------- Latest Section ------------------------ */
+
+.latest {
+  padding: 6rem 0;
+}
+
+.single-latest {
+  width: 25.7rem;
+  margin-bottom: 3rem;
+}
+
+.single-latest:not(:last-child) {
+  margin-right: 3.7rem;
+}
+
+.latest-container {
+  flex-direction: column;
+}
+
+.lates-top {
+  margin-bottom: 3rem;
+}
+
+.lates-top,
+.latest-bottom {
+  flex-wrap: wrap;
+}
+
+.latest-img-box {
+  width: 100%;
+  height: 20rem;
+}
+
+.latest-img-box img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.latest-title {
+  margin-bottom: -1rem;
+}
+
+.single-latest h3 {
+  margin: 1rem 0;
+}
+
+.single-latest p {
+  font-size: 1.6rem;
+  color: rgba(0, 0, 0, 0.8);
+}
+
+.single-latest p span {
+  font-size: 1.2rem;
+  color: var(--color-gray-span);
+  margin-right: 1rem;
+  text-decoration: line-through;
+  font-weight: 700;
+}
+
+/* --------------------Footer------------------------- */
+
+.footer {
+  margin-bottom: -5rem;
+}
+
+.footer-top {
+  background-color: var(--color-header);
+  padding: 4.2rem 0 2.8rem 0;
+}
+
+.footer-icons {
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-icons i::after {
+  content: "";
+  position: absolute;
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  background-color: rgba(255, 255, 255, 0.1);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: rgba(255, 255, 255, 0.1) 2px solid;
+  z-index: 3;
+}
+
+.footer-bottom {
+  background-color: black;
+  padding: 3.9rem 0;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-bottom p {
+  font-size: 1.4rem;
+  color: rgba(255, 255, 255, 0.6);
+  margin-left: 0.5rem;
+}
+
+.footer-links {
+  justify-content: center;
+  align-items: center;
+  margin-top: 1.5rem;
+}
+
+/* ----------------- Hover Styles ----------------- */
+
+.navbar-nav__link:hover {
+  color: rgba(0, 0, 0, 0.7) !important;
+}
+
+.footer-links a:hover {
+  color: white;
+}
+
+.footer-icons i:hover::after {
+  cursor: pointer;
+}
+
+.single-latest a:hover {
+  cursor: pointer;
+  color: orangered;
+  text-decoration: underline;
+}
+
+.single-news a:hover {
+  text-decoration: underline;
+  color: orangered;
+}
+
+.header-icons a:hover {
+  color: white;
+}
+
+.toggle__link:hover {
+  z-index: 400;
+  width: 100%;
+  color: rgba(0, 0, 0, 0.7) !important;
+}
+
+.search-btn i:hover {
+  color: rgba(0, 0, 0, 0.7);
+}
+
+/* ----------------- Sticky Button ----------------- */
+
+.sticky-button {
+  position: sticky;
+  left: calc(100% - 7rem);
+  bottom: 3rem;
+  width: 5rem;
+  height: 5rem;
+  background-color: #146ff8;
+  border-radius: 50%;
+  box-shadow: 0 0 3rem rgba(0, 0, 0, 0.2);
+}
+
+/* ----------------- Media Queries ----------------- */
+
+@media only screen and (max-width: 1150px) {
+  .main {
+    width: 90%;
+  }
+
```

Comment:
## Media queries

CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))

* No small break point as it is mobile first and the base styles are for the mobile screen. ([Screenshot on mobile first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
* medium: min-width around 768px
* large: min-width 1024px

- [ ] Please remove all media queries you have on the entire page (CSS file) and work with these ones

```CSS
/* medium screens */
media only screen and (min-width:768 px) {
}

/* large screens */
media only screen and (min-width:1024 px) {
}

```